pkg/stats/bin-width.go

```diff
@@ -0,0 +1,18 @@
+package stats
+
+import "find-bin-width/pkg/xtype"
+
+func findBinWidth(dataType xtype.DataType, data any) Stats {
```

Comment:
i think this function and its friends should not be called `findBinWidth` (really, any more than the script should be called that).   

it could be called findSummaryStats or something, among which bin width is a single stat

Reply:
Updated.